src/articleView-components/CommentsContainer.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getCommentsByArticleId } from "../api";
+import { Comment } from "./Comment";
+
+export function CommentsContainer({ article }) {
+  const [comments, setComments] = useState([]);
+  const { article_id } = useParams();
```

Comment:
If you've passed the entire article in on props, do you need to useParams here? One or the other would be ideal.